packages/api-bindings/codegen-api.yml

```diff
@@ -75,6 +78,24 @@ generates:
           skipTypename: true
           immutableTypes: false
           inlineFragmentTypes: combine
-      - 'typescript-react-apollo'
+          omitOperationSuffix: true
+          operationResultSuffix: 'Data'
+
+  src/graphql/hooks.ts:
+    plugins:
+      - add:
+          content: [
+              '/* eslint-disable import/no-default-export */', # generatedTypes default export
+              '/* eslint-disable @typescript-eslint/no-explicit-any */', # see extensive use of FieldPolicy<any> | FieldReadFunction<any>
+              '/* eslint-disable @typescript-eslint/naming-convention */', # interface PossibleTypesResultData not starting with I
+              '/* eslint-disable no-restricted-imports */', # import * from @apollo/client
+              "import * as Operations from './operations';",
+            ]
+      - 'typescript-react-apollo':
+          pureMagicComment: true # keep
+          importOperationTypesFrom: 'Operations'
+          omitOperationSuffix: true # MUST goes with typescript-operations.omitOperationSuffix in src/graphql/operations.ts
```

Comment:
I guess this is the main change in this PR outside of renaming.
anything else I should be aware of?

Reply:
Correct, this is the core of the change. Everything else is in support or a consequence of the change.